new/prusti-specs/src/specifications/preparser.rs

```diff
@@ -0,0 +1,637 @@
+/// The preparser splits a composite Prusti assertion into atomic assertions, parses the resulting
+/// Rust expressions, and then assembles the composite Prusti assertion.
```

Comment:
```suggestion
/// Rust expressions, and then assembles the composite Prusti assertion. For a more high-level overview, please check `mod.rs`.
```

(Do not forget to format the documentaiton lines to be no longer than 80 characters.)